generators/app/src/techs.js

```diff
@@ -19,7 +19,8 @@ module.exports = function (GulpAngularGenerator) {
       this.props.foundationComponents.key,
       this.props.cssPreprocessor.key,
       this.props.jsPreprocessor.key,
-      this.props.htmlPreprocessor.key
+      this.props.htmlPreprocessor.key,
+      this.props.protractorFramework.key
```

Comment:
By adding a new item to the prompt, this causes an error (since `this.props.protractorFramework` is `undefined`)  when the user loads an existing .yo.rc. 

How have you historically managed changes like this?